app/controllers/middleware_server_controller.rb

```diff
@@ -8,13 +8,42 @@ class MiddlewareServerController < ApplicationController
   after_action :set_session_data
 
   OPERATIONS = {
-    :middleware_server_reload => {:op   => :reload_middleware_server,
-                                  :hawk => N_('Not reloading Hawkular server'),
-                                  :msg  => N_('Reload initiated for selected server(s)')
+    :middleware_server_reload  => {:op    => :reload_middleware_server,
+                                   :skip  => true,
+                                   :hawk  => N_('reloading'),
```

Comment:
The message if the Hawkular server was part of the selection (in the list case).